src/model/client.ts

```diff
@@ -338,6 +442,35 @@ export class CommandoClient extends Client {
     return this.runCommand(msg, argsObject)
   }
 
+  /**
+   * Tests a command's arguments.
```

Comment:
What does it mean to "test a commands arguments"? What are we testing them against? How are we testing them?

Can this be more descriptive?